code/cellular_automaton/src/elementary_cellular_automata/ElementaryCellularAutomaton.java

```diff
@@ -65,11 +65,15 @@ private int getCellIndex(int cell) {
 	
 	@Override
 	public String toString() {
-		StringBuilder builder = new StringBuilder();
+		String result = "";
 		boolean[] array = grid[step];
 		for (int i = 0; i < array.length; i++) {
-		    if (array[i]) builder.append("1"); else builder.append("0");
+		    if (array[i]) {
+				result += "1";
+			} else {
```

Comment:
Fix the indentation here. Also consider the ternary operator for terseness.

Reply:
My identation at line 72 is correct...
![screenshot](https://user-images.githubusercontent.com/23243382/43037593-06b642c0-8d0f-11e8-90b3-a86816acb1a3.png)